### DISCUSSION TITLE
Bitbucket support

### DISCUSSION THREAD

**Discussion by IanBoyte** [VictorGrycuk/streamdeck-repository-watcher#14] [Ideas]
I wonder how different the Bitbucket api is to the github api?

**Comment by VictorGrycuk** [comment]
As long as the Bitbucket API expose the same information, it should be possible to integrate.
I would like to add support for Bitbucket and GitLab, but at the moment this plugin is in need of a rewrite.
Once I am done fixing bugs from my other plugins, I would like to work on this one, make it cleaner, and if possible, easier to use.
